package.json

```diff
@@ -48,10 +48,10 @@
     "@babel/preset-env": "^7.11.0",
     "@babel/traverse": "^7.11.5",
     "@rollup/plugin-babel": "^5.1.0",
-    "@rollup/plugin-commonjs": "^11.0.0",
-    "@rollup/plugin-json": "^4.0.0",
+    "@rollup/plugin-commonjs": "^15.1.0",
+    "@rollup/plugin-json": "^4.1.0",
```

Comment:
[v4.0.3](https://github.com/rollup/plugins/blob/master/packages/json/CHANGELOG.md#v403) is the first to add it in the peerDep range, hence this change. Older versions are forward-compatible but will produce a peerDep warning